packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -807,6 +807,36 @@ export default class Meetings extends WebexPlugin {
     );
   }
 
+  /**
+   * Creates a noise reduction effect
+   *
+   * @param {Object} options optional custom effect options
+   * @returns {Promise<effect>} noise reduction effect.
+   * @public
+   * @memberof Meetings
+   */
+  createNoiseReductionEffect = async (options = {}) => {
+    // @ts-ignore
```

Comment:
What complaint from ts are we ignoring here?

Reply:
1. previously I wrote a method where we were **creating the effect** and was **applying it on the passed audio/video track** in the same method and also **enabling it** at the end of the method.

But after discussion with Arun/Marcin we came to a conclusion that this is not the approach we planned to take instead just **return the effect** and let developers use it on their side of code. We will create a detailed wiki explaining the steps to use these steps on the tracks. I updated the code as per the discussed approach.

2. for using @ts-ignore, we are ignoring the warning which is `Property 'webex' does not exist on type 'Meetings'`